README.md

```diff
@@ -5,52 +5,81 @@
 
 Self-contained fat-jar micro-service to publish court information
 
-### Pre-requisite
+## Pre-requisite
 
-`Docker` Even when running the tests docker is used by the integration test to load `localstack` (for AWS services). The build will automatically download and run `localstack` on your behalf.
+`Docker` Even when running the tests docker is used to start `localstack` (for AWS services) and a Postgres database. The build will automatically download and run these containers on your behalf.
 
-### Building
+## Building
 
 ```./gradlew build```
 
-### Running
+## Running
```

Comment:
Tidied up the instructions around running the app as they didn't work and had been copied into other applications.